Gemfile

```diff
@@ -24,6 +24,11 @@ end
 
 gem 'sqlite3', '~> 1.3.6'
 
+# Version 6.0.4 of rdoc requires Ruby 2.2
+if RUBY_VERSION < '2.2'
+  gem 'rdoc', '< 6.0.4'
+end
```

Comment:
If rdoc is failing it's likely down in the app installs, not in our outer Gemfile, you can see in the example app generator gsub rules for generated Gemfiles which help to mitigate this.